components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtDBQueries.java

```diff
@@ -195,4 +198,16 @@
     public static final String CHECK_AVAILABILITY_OF_IDN_CERTIFICATE_TABLE_ORACLE = "SELECT ID FROM " +
             "IDN_CERTIFICATE WHERE ROWNUM < 2";
 
+    public static final String ADD_SP_CONSENT_PURPOSE = "INSERT INTO SP_CONSENT_PURPOSE (APP_ID, PURPOSE_ID, " +
```

Comment:
Shall we make this table SP_CONSENT_PURPOSE_ASSOC since this table is used to maintain an association.